### PR TITLE
Bobakm patch 2

### PR DESCRIPTION
**Pull request by bobakm**
Updated two pdfs

vodrahalli23a.pdf
zhou23a.pdf

where references and figures were not appearing correctly

### CODE REVIEW

**Comment by bobakm** [comment]
Updating zhou23a.pdf and vodrahalli23a.pdf to fix broken citations/references

**Review by lawrennd — CHANGES_REQUESTED** [review]
HI Bobak, I think the vodrahalli23 file should be called vodrahalli23a.pdf ...

**Comment by bobakm** [comment]
Updated file name in folder directly to vodrahalli23a.pdf

File: vodrahalli23a/Vodrahalli23.pdf
Comment (bobakm) [inline vodrahalli23a/Vodrahalli23.pdf:1]: name changed in folder structure

**Review by bobakm — COMMENTED** [review]
[vodrahalli23a.pdf](https://github.com/mlresearch/v209/files/11789766/vodrahalli23a.pdf)

File: vodrahalli23a/Vodrahalli23.pdf
Comment (lawrennd) [reply vodrahalli23a/Vodrahalli23.pdf:1]: HI Bobak, Are you sure you've updated the pull request with that change? I don't see it here.

Comment (bobakm) [reply vodrahalli23a/Vodrahalli23.pdf:1]: Hi - you seem to be right both the old file, named Vodrahalli23.pdf and the new one vodrahalli23a.pdf added in response to these comments. would it make sense to delete the request/fork and re-do it? I can't seem to go "replace" files

Comment (lawrennd) [reply vodrahalli23a/Vodrahalli23.pdf:1]: Yes, likely easier to delete and send a new one.

**Pull request closed without merging**